test/components/playground/SelectedPolicy.spec.js

```diff
@@ -15,44 +15,101 @@
  */
 
 import React from "react";
-import { render, screen } from "@testing-library/react";
+import { render, screen } from "test/testing-utils/renderer";
 import SelectedPolicy from "components/playground/SelectedPolicy";
+import { copy } from "utils/shared-utils";
+import userEvent from "@testing-library/user-event";
+
+jest.mock("utils/shared-utils");
 
 describe("SelectedPolicy", () => {
-  let policy, rerender;
+  let policy, setPolicy, clearEvaluation;
 
   beforeEach(() => {
-    policy = {
-      name: chance.string(),
-      description: chance.string(),
-      regoContent: chance.word({ syllables: chance.d10() }),
-    };
-
-    const utils = render(<SelectedPolicy policy={policy} />);
-    rerender = utils.rerender;
+    setPolicy = jest.fn();
+    clearEvaluation = jest.fn();
   });
 
-  it("should render the rego policy label", () => {
-    expect(screen.getByText(/rego policy code/i)).toBeInTheDocument();
+  afterEach(() => {
+    jest.resetAllMocks();
   });
 
-  it("should render the rego policy code", () => {
-    expect(screen.getByTestId("regoPolicyCode")).toBeInTheDocument();
-    expect(
-      screen.getByText(policy.regoContent, { exact: false })
-    ).toBeInTheDocument();
-  });
+  describe("policy has not been selected", () => {
+    beforeEach(() => {
+      policy = null;
+      render(
+        <SelectedPolicy
+          policy={policy}
+          setPolicy={setPolicy}
+          clearEvaluation={clearEvaluation}
+        />
+      );
+    });
+
+    it("should render the instructions to search for a policy", () => {
+      expect(
+        screen.getByText(/search for a policy to begin/i)
+      ).toBeInTheDocument();
+    });
 
-  it("should render the policy name", () => {
-    expect(screen.getByText("Policy")).toBeInTheDocument();
-    expect(screen.getByText(policy.name)).toBeInTheDocument();
+    it("should render the search button", () => {
+      expect(
+        screen.getByLabelText(/^search for policies/i)
+      ).toBeInTheDocument();
+    });
   });
 
-  it("should render the instructions if no policy is selected", () => {
-    rerender(<SelectedPolicy policy={null} />);
+  describe("policy has been selected", () => {
+    beforeEach(() => {
+      policy = {
+        name: chance.string(),
+        description: chance.string(),
+        regoContent: chance.word({ syllables: chance.d10() }),
+      };
+      render(
+        <SelectedPolicy
+          policy={policy}
+          setPolicy={setPolicy}
+          clearEvaluation={clearEvaluation}
+        />
+      );
+    });
+
+    it("should render the rego policy label", () => {
```

Comment:
```suggestion
    it("should render the Rego policy label", () => {
```
Definitely a nitpick, but I think Rego is always capitalized. Same thing for a few of the other test titles.